sda-commons-server-dropwizard/src/main/java/org/sdase/commons/server/dropwizard/bundles/LookupTracker.java

```diff
@@ -0,0 +1,12 @@
+package org.sdase.commons.server.dropwizard.bundles;
+
+import java.util.Set;
+
+/** Tracks keys that where looked up in the environment. */
```

Comment:
nit: "that **were** looked up"

Reply:
fixed … guess that's my most common mistake in writing